tests/__init__.py

```diff
@@ -6,3 +6,7 @@
 
 # credentials count after post-processing
 SAMPLES_POST_CRED_COUNT: int = 18
+
+# well known string with all latin letters
+AZ_DATA = b"The quick brown fox jumps over the lazy dog"
+AZ_STRING = AZ_DATA.decode(encoding="ascii")
```

Comment:
TEST_DATA is more common name. Probably it might be used in specific test cases.
The pangram (a-z letters) is well known and their hash might be found in many references.
When the bytes are written then hash of  file must be same like in the references.  
Sometime with text working may be mistakes like:
C/C++ - \0 is written in tail
bash - \n appended in echo case ...